src/Sarif.FunctionalTests/SarifConverterTestData/FxCop/FxCopExampleReport2.xml.sarif

```diff
@@ -17,20 +17,60 @@
           }
         ]
       },
+      "logicalLocations": {
+        "SecurityCryptographyRuleTests": [
+          {
+            "name": "SecurityCryptographyRuleTests",
+            "kind": "namespace"
+          }
+        ],
+        "securitycryptographyruletests.dll": [
+          {
+            "name": "securitycryptographyruletests.dll",
+            "kind": "module"
+          }
+        ],
+        "SecurityCryptographyRuleTests.DESCannotBeUsed": [
+          {
+            "name": "securitycryptographyruletests.dll",
+            "kind": "module"
+          },
+          {
+            "name": "SecurityCryptographyRuleTests",
+            "kind": "namespace"
+          },
+          {
+            "name": "DESCannotBeUsed",
+            "kind": "type"
```

Comment:
not in spec. this should be 'class' or 'struct'. if fxcop allows for differentiation